pkg/cstor/cstor.go

```diff
@@ -517,7 +517,7 @@ func (p *Plugin) getScheduleName(backupName string) string {
 	// If it is scheduled backup then we need to get the schedule name
 	splitName := strings.Split(backupName, "-")
 	if len(splitName) >= 2 {
-		t, err := time.Parse("20060102150405", splitName[len(splitName)-1])
+		_, err := time.Parse("20060102150405", splitName[len(splitName)-1])
```

Comment:
What is this hardcoding?

Reply:
It is time format used by golang. https://golang.org/pkg/time/#pkg-constants

Reply:
Sometimes go drives me nuts!